test/behaviours/feedback.spec.js

```diff
@@ -0,0 +1,271 @@
+'use strict';
+
+/* eslint max-nested-callbacks: 0 */
+
+const proxyquire = require('proxyquire');
+
+const mockUtils = {
+  sendEmail: sinon.stub(),
+  pollEmailStatus: sinon.stub()
+};
+
+const mockConfig = {
+  notify: {
+    templateFeedback: 'templateFeedback',
+    feedbackEmail: 'feedbackEmail',
+    feedbackEmailSessionName: 'feedback-email',
+    statusRetryInterval: 1000
+  }
+};
+
+const mockUUID = {
+  v4: () => 'mockUUID'
+};
+
+const Behaviour = proxyquire('../../apps/feedback/behaviours/feedback', {
+  '../../../lib/utils': mockUtils,
+  '../../../config': mockConfig,
+  'uuid': mockUUID
+});
+
+describe('Feedback behaviour', () => {
+
+  let req;
+  let res;
+  let Feedback;
+  let testInstance;
+  let nextStub;
+
+  beforeEach(() => {
+    req = {
+      form: {
+        values: {}
+      },
+      log: sinon.stub(),
+      sessionModel: {
+        get: sinon.stub(),
+        set: sinon.stub()
+      },
+      session: {
+        save: sinon.stub()
+      },
+      headers: {
+        referer: ''
+      }
+    };
+
+    res = {};
+  });
+
+  describe('getValues', () => {
+
+    class Base {
+      getValues() {}
+    }
+
+    beforeEach(() => {
+      Feedback = Behaviour(Base);
+      testInstance = new Feedback();
+      nextStub = sinon.stub();
+    });
+
+    describe('on redirect to feedback page', () => {
+
+      it('should set feedbackReturnTo to the location where the user directed to the feedback page from', () => {
+        req.headers.referer = 'referred-from-link.com';
+        const expected = 'referred-from-link.com';
```

Comment:
set a constant called something like `referredFrom` and use it in all three places this string is required